packages/dart_firebase_admin/lib/src/utils/jwt.dart

```diff
@@ -9,14 +9,28 @@ class EmulatorSignatureVerifier implements SignatureVerifier {
   @override
   Future<void> verify(String token) async {
     // Signature checks skipped for emulator; no need to fetch public keys.
+
     try {
-      verifyJwtSignature(
```

Comment:
I would recommend maybe keeping the `verifyJwtSignature` function call, and modify the function to only catch `JWTExpiredException`, since the handling of `JWTExpiredException` is the same in both verifiers.